### ISSUE TITLE
Question about training the model

### ISSUE THREAD

**Issue by limt15** [XinyiZ001/CapsGNN#3]
Hi, Xinyi
Thank you very much for sharing your code on github!
I had run the code followed by the steps listed in README. But, when I trained the model by `python3 main.py --dataset_dir data_plk/ENZYMES --epochs 3000 --lambda_val 0.5` 
the training program is stuck in an infinite loop, which means that, when training to step 2999, the program will start training from 0 again, it cannot stop the training at the step 3000.
What wrong with it?
Can you help me to solve this problem?
Looking forward for hearing from you!

**Comment by XinyiZ001** [comment]
Hi,

This code is used for experiments, so it will automatically run 10-fold cross-validation which means it will run 10 times for each dataset and record their accuracy.

This setting is done through these lines:
```
71    for train_test_split_groups_idx,groups_dict in enumerate(train_test_split_groups):
72         gd.graphs_dataset_train = groups_dict['train']
73         gd.graphs_dataset_valid = groups_dict['val']
74         gd.graphs_dataset_test = groups_dict['test']
```
I hope this is helpful for you.
Please feel free to contact at us if you still have some questions.
